LeteSansMath/mathfonts.css

```diff
@@ -0,0 +1,41 @@
+/*
+Lete Sans Math is released under the SIL Open Font License. See the files in this
+directory for details. The font can be obtained from several TeX distributions
+or package managers. It is recommended to use it with the Lato font or other
+sans-serif fonts for the surrounding text.
+
+The OTF fonts have been obtained from http://mirrors.ctan.org/fonts/ and
+converted to WOFF.
+*/
+
+@namespace url('http://www.w3.org/1999/xhtml');
+@namespace m url('http://www.w3.org/1998/Math/MathML');
+
+@font-face {
+    font-family: Lete Sans Math;
+    src: local('Lete Sans Math'), local('Lete-Sans-Math'),
+         url('LeteSansMath.woff2'), url('LeteSansMath.woff');
+}
+
+.htmlmathparagraph, m|mtext {
+    font-family: Lete Sans Math;
+}
+m|math {
+    font-family: Lete Sans Math;
+}
+
+.oldstylenumbers {
+    font-variant-numeric: oldstyle-nums;   /* Recommended syntax */
+    font-feature-settings: 'onum';         /* OpenType feature syntax */
+    -moz-font-feature-settings: 'onum';    /* Gecko <= 33 syntax */
+    -webkit-font-feature-settings: 'onum'; /* Blink syntax */
+}
+
+@font-feature-values Lete Sans Math { @stylistic { calligraphic: 1; } }
+m|*.calligraphic {
+    font-variant-alternates: stylistic(calligraphic); /* Recommended syntax */
+    font-feature-settings: 'salt';         /* OpenType feature syntax */
+    -moz-font-feature-settings: 'salt';    /* Gecko <= 33 syntax */
+    -webkit-font-feature-settings: 'salt'; /* Blink syntax */
+}
+
```

Comment:
And this?

Reply:
Ah, indeed, that is incorrect. The available features are:
```
$ otfinfo -f LeteSansMath.otf 
cv11    <unknown feature>
kern    Kerning
onum    Oldstyle Figures
ss06    Stylistic Set 6
ss08    Stylistic Set 8
ss09    Stylistic Set 9
ssty    Math Script Style Alternates
```

Interestingly, the TeX source code offers even more SS - something seems to be wrong here:
```
\newopentypefeature{Style}{leqslant}{+ss04}
\newopentypefeature{Style}{smaller}{+ss05}
\newopentypefeature{Style}{subsetneq}{+ss06}
\newopentypefeature{Style}{parallelslant}{+ss07}
\newopentypefeature{Style}{upint}{+ss08}
\newopentypefeature{Style}{upneg}{+ss09}
```

Should the `calligraphic` font feature values simply be removed from the CSS file?

Reply:
Yes, I suppose this is what we should do to avoid misleading users on unsupported features. That's what seems to be done for https://github.com/fred-wang/MathFonts/blob/main/FiraMath/mathfonts.css for example.

Reply:
Done, removed now.

Reply:
I also reported the discrepancy between the LaTeX sty file and the actual font features here https://github.com/abccsss/LeteSansMath/issues/12